### ISSUE TITLE
Help import users from ldap group

### ISSUE THREAD

**Issue by TheAlex88** [nuxsmin/sysPass#1544]
Hello,

I am trying to import users from an Active directory group but I cannot.

If I import users within that organizational unit or the same group, I do, but the users of that group do not.

I have the latest version of syspass installed.

Can somebody help me?

Thank you very much in advance.

Greetings.

**Comment by daveworthley** [comment]
Trying selecting server type Standard instead of Active Directory. That worked for me? Although doesn't make sense? :-)

**Comment by TheAlex88** [comment]
Hello,

I have tried different types of server and it does the same to me.

If I do the search directly in the organizational unit, it detects the users and imports them from me, but if I import the group, it also matters to me, but the users still don't appear.

I have tried when I have imported the group, to login equally with the users that are in the group but it says wrong login.

Something I'm doing wrong and I don't understand.

Thank you.

Greetings.

**Comment by TheAlex88** [comment]
Hello,

I have tried different types of server and it does the same to me.

If I do the search directly in the organizational unit, it detects the users and imports them from me, but if I import the group, it also matters to me, but the users still don't appear.

I have tried when I have imported the group, to login equally with the users that are in the group but it says wrong login.

Something I'm doing wrong and I don't understand.

Thank you.

Greetings.

**Comment by TheAlex88** [comment]
Hello,

I have tried different types of server and it does the same to me.

If I do the search directly in the organizational unit, it detects the users and imports them from me, but if I import the group, it also matters to me, but the users still don't appear.

I have tried when I have imported the group, to login equally with the users that are in the group but it says wrong login.

Something I'm doing wrong and I don't understand.

Thank you.

Greetings.
